src/System.Net.Sockets/tests/FunctionalTests/SocketOptionNameTest.cs

```diff
@@ -14,63 +14,42 @@ namespace System.Net.Sockets.Tests
 {
     public class SocketOptionNameTest
     {
-        private static bool SocketsReuseUnicastPortSupport
-        {
-            get
-            {
-                return Capability.SocketsReuseUnicastPortSupport().HasValue &&
-                    Capability.SocketsReuseUnicastPortSupport().Value;
-            }
-        }
-
-        private static bool NoSocketsReuseUnicastPortSupport
-        {
-            get
-            {
-                return Capability.SocketsReuseUnicastPortSupport().HasValue &&
-                    !Capability.SocketsReuseUnicastPortSupport().Value;
-            }
-        }
-
-        [OuterLoop] // TODO: Issue #11345
-        [ConditionalFact(nameof(NoSocketsReuseUnicastPortSupport))]
-        public void ReuseUnicastPort_CreateSocketGetOption_NoSocketsReuseUnicastPortSupport_Throws()
-        {
-            var socket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
-
-            Assert.Throws<SocketException>(() =>
-                socket.GetSocketOption(SocketOptionLevel.Socket, SocketOptionName.ReuseUnicastPort));
-        }
+        private static bool SocketsReuseUnicastPortSupport => Capability.SocketsReuseUnicastPortSupport().HasValue;
```

Comment:
What behavior change are you referring to?  The check for the value (if it has one) is now done in the test instead, to avoid having two tests, one of which will always be skipped.